app/javascript/src/components/Player.tsx

```diff
@@ -202,6 +230,89 @@ class Player extends React.Component<PlayerProps, PlayerState> {
     }
   }
 
+  private renderDebugger() {
+    return this.props.debug ? (
+      <div>
+        <div className="d-helper"></div>
+        <div className="debugger">
+          <div className="d-section">
+            <h4><u>Current Items</u></h4>
+            <ul className="d-list">
+              {this.state.currentItems.map((item, i) => {
+                return (
+                  <li key={item.concat(i.toString())}>
+                    <div>
+                      <label
+                        className="sr-only"
+                        htmlFor={`name-for-${item}`}
+                      >
+                        Item Name
+                      </label>
+                      <input
+                        id={`name-for-${item}`}
+                        onBlur={this.setItem.bind(this, i)}
+                        defaultValue={item}
+                      />
```

Comment:
thoughts on turning this into a select input so users could pick from the list of possible items instead of having to type it out themselves?

Reply:
That is probably a better idea. Will prevent spelling mistakes and the like, so I will do that.

Reply:
I will also remove the url hashing.